momento-sdk/src/main/java/momento/sdk/SubscriptionWrapper.java

```diff
@@ -175,6 +186,15 @@ private void handleSubscriptionItemMessage(_SubscriptionItem item) {
     _TopicValue topicValue = topicItem.getValue();
     options.subscriptionState.setResumeAtTopicSequenceNumber(
         (int) topicItem.getTopicSequenceNumber());
+    options.subscriptionState.setResumeAtTopicSequencePage((int) topicItem.getSequencePage());
```

Comment:
given recent discussion about these types in dotnet, is `int` the type we want here? should it be a `long` or something instead to more closely match the uint64 type in the protos?

Reply:
Yeah, I wanted to discuss that in mid-week checkin meeting. We were originally casting the sequence number to int, and therefore I kept casting to it. But based the latest discussions, it makes sense to case them to long